crates/forge/tests/cli/test_cmd.rs

```diff
@@ -137,16 +136,11 @@ contract ATest is DSTest {
     .unwrap();
 
     cmd.arg("test").assert_success().stdout_eq(str![[r#"
-[COMPILING_FILES] with [SOLC_VERSION]
-[SOLC_VERSION] [ELAPSED]
-Compiler run successful!
-
+...
```

Comment:
why was this changed? again the point of all these substitution is so that if the test output changes you can just run the test suite once with SNAPSHOTS=overwrite and it will be updated normally. now you have to do that and also re-add manually `...` since those are manual filters

Reply:
yep, reverted, please check (got wrong the `SNAPSHOTS=overwrite` usage but now clear, thank you)